analytic_base_department/tests/test_line_department.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2016 Yannick Vaucher (Camptocamp)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.tests.common import TransactionCase
+
+
+class LineDepartmentCase(TransactionCase):
+
+    def test_default_department(self):
+        """In a new users form, a user set only the firstname."""
+        aal = self.env['account.analytic.line'].sudo(self.user).new()
+        department_id = aal.default_get(['department_id']).get('department_id')
+        self.assertEqual(department_id, self.dep.id)
+
+    def setUp(self):
```

Comment:
Can you place method `def setUp()` before method `def test_default_department()` ?